### DISCUSSION TITLE
Questions about solid start

### DISCUSSION THREAD

**Discussion by MadadiSaeed** [solidjs/solid#1311] [Q&A] [answered]
Hi. I have questions about solid start. 
1. Some of the APIs have dollar sign in the end of their names. Why is that? Is it necessary?
2. Why not use `@solidjs/start` package name instead of `solid-start` since it is an official library?
3. Are `entry-server.tsx` & `entry-client.tsx` configurable?

**Comment by ryansolid** [comment, marked as answer]
1. This is to indicate functions that are compiled and hoist out of the execution context. Using a convention may allow us to let people write their own $ functions in the future.
2. This is fair and I'd like to consider this. Right now we use a lot of sub exports like `solid-start/server` and `@solidjs/start/server` seems a bit much perhaps. I also got the `@solidstart` namespace on npm so that is an option too.
3. I believe so. We do something like this in the Docs project that is at the root of the `solid-start` project.

**Reply by MadadiSaeed** [reply]
Thanks for the response. By the way I asked the first one in your recent stream.
